test/index.ts

```diff
@@ -0,0 +1,9 @@
+import {Scanner} from '../src/scanner';
+import {Rule} from '../src/rules';
+
+let rules: Rule[];
```

Comment:
maybe needs cleanup? Seems like this never gets assigned - and global state in tests that's not handled in test fixture may go sideways in general